### DISCUSSION TITLE
maestral tries to sync ignored socket

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [samschott/maestral#793] [Q&A] [answered]
When I open a .R file in RStudio, RStudio creates in the working directory a socket named "rstudio=", which maestral ties to sync.

I have put "rstudio=" in `.mignore` but maestral is still trying to sync it.

How can I stop it from syncing?

**Comment by samschott** [comment, marked as answer]
If you want to ignore all items named "rstudio=", no matter which directory they live in, you'll need to put `**/rstudio=` in the mignore file. I've followed the specification from gitignore at https://git-scm.com/docs/gitignore.

Let me know if this works, otherwise its a bug...

**Reply by raffaelemancuso** [reply]
> If you want to ignore all items named "rstudio=", no matter which directory they live in, you'll need to put `**/rstudio=` in the mignore file. I've followed the specification from gitignore at https://git-scm.com/docs/gitignore.
> 
> Let me know if this works, otherwise its a bug...

Right, thanks.

It looks like the "=" at the end of the name is not part of the filename itself, it is just a symbol appended by ls which means that the file is a socket (see [here](https://stackoverflow.com/questions/31594069/equals-sign-at-end-of-filename) and [here](https://www.gnu.org/software/coreutils/manual/html_node/General-output-formatting.html)).

Putting `**/rstudio` in `.mignore` indeed works and maestral does not try to sync the file anymore.

Although in the logs it catches the file creation event, it is later filtered out. It may be useful to have an entry that log the reason why it's being filtered out?
